### PR TITLE
driver hamamatsurx: Migrate to using bytes for Python 3

### PR DESCRIPTION
**Pull request by pieleric**
In Python 3, the network sockets send and receive bytes.

Also adjust a couple of extra places in the message handling.
In particular, the CameraInfo now behaves a little bit less special, and
each line is return as a separate return value instead of one value
containing a list.

These changes haven't be tested on the hardware yet.

### CODE REVIEW

File: src/odemis/driver/hamamatsurx.py [hunk 1]
```diff
@@ -1157,10 +1160,10 @@ def readCommandResponse(self):
                         continue  # return to try-statement and start receiving again
 
                     if self._getReadoutCamInfo:
-                        # command parent.CamParamGet("Setup", "CameraInfo") behaves differently then all other commands.
+                        # HACK: command CamParamGet("Setup", "CameraInfo") behaves differently then all other commands.
```
Comment (Sabrina85) [inline src/odemis/driver/hamamatsurx.py]: I think I made a language error here. Think it should be `...from all the other commands`

**Comment by Sabrina85** [comment]
Will Iheb test these changes?

File: src/odemis/driver/hamamatsurx.py (hunk 1 above)
Comment (tepals) [reply src/odemis/driver/hamamatsurx.py]: just a very small note: different than is not always incorrect as long as you spell it with an "a" and not an "e". https://www.grammar.com/different-from-vs-different-than/

**Comment by pieleric** [comment]
Fixed the spelling.

**Comment by pieleric** [comment]
> Will Iheb test these changes?
I'll let him know that next update should be double-checked. However, I'm not in a rush to test the Python 3 support. For now, I'd just check that it still works in Python 2.

**Pull request merged**